src/overlays/actors/ovl_En_Demo_heishi/z_en_demo_heishi.h

```diff
@@ -7,12 +7,46 @@ struct EnDemoheishi;
 
 typedef void (*EnDemoheishiActionFunc)(struct EnDemoheishi*, GlobalContext*);
 
+typedef enum {
+    /*  0 */ DEMOHEISHI_LIMB_NONE,
+    /*  1 */ DEMOHEISHI_LIMB_ROOT,
+    /*  2 */ DEMOHEISHI_LIMB_PELVIS,
+    /*  3 */ DEMOHEISHI_LIMB_LEFT_THIGH,
+    /*  4 */ DEMOHEISHI_LIMB_LEFT_CALF,
+    /*  5 */ DEMOHEISHI_LIMB_LEFT_FOOT,
+    /*  6 */ DEMOHEISHI_LIMB_RIGHT_THIGH,
+    /*  7 */ DEMOHEISHI_LIMB_RIGHT_CALF,
+    /*  8 */ DEMOHEISHI_LIMB_RIGHT_FOOT,
+    /*  9 */ DEMOHEISHI_LIMB_TORSO,
+    /* 10 */ DEMOHEISHI_LIMB_LEFT_SHOULDER,
+    /* 11 */ DEMOHEISHI_LIMB_LEFT_FOREARM,
+    /* 12 */ DEMOHEISHI_LIMB_LEFT_HAND,
+    /* 13 */ DEMOHEISHI_LIMB_RIGHT_SHOULDER,
+    /* 14 */ DEMOHEISHI_LIMB_RIGHT_FOREARM,
+    /* 15 */ DEMOHEISHI_LIMB_RIGHT_HAND_WITH_SPEAR,
+    /* 16 */ DEMOHEISHI_LIMB_HEAD,
+    /* 17 */ DEMOHEISHI_LIMB_MAX
+} EnDemoheishiLimbs;
+
 typedef struct EnDemoheishi {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x110];
-    /* 0x0254 */ EnDemoheishiActionFunc actionFunc;
-    /* 0x0258 */ char unk_258[0x68];
-} EnDemoheishi; // size = 0x2C0
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ Vec3s jointTable[DEMOHEISHI_LIMB_MAX];
+    /* 0x1EE */ Vec3s morphTable[DEMOHEISHI_LIMB_MAX];
+    /* 0x254 */ EnDemoheishiActionFunc actionFunc;
+    /* 0x258 */ s16 headRotY;
+    /* 0x25A */ s16 headRotX;
+    /* 0x25C */ s16 headRotZ;
+    /* 0x25E */ s16 headRotYTarget;
+    /* 0x260 */ s16 headRotXTarget;
+    /* 0x262 */ s16 headRotZTarget;
+    /* 0x264 */ s32 animIndex;
+    /* 0x268 */ s16 timer;
+    /* 0x26C */ f32 frameCount;
+    /* 0x270 */ s16 unk270;     // set and not used
```

Comment:
```suggestion
    /* 0x270 */ s16 isTalking;     // set and not used
```

Reply:
I asked about this, but EllipticEllipsis suggested keeping it unnamed since it's isn't used anywhere. Should I make your change or keep his suggestion?

Reply:
I'll talk to him and we can figure it out. Thankyou.

Reply:
Should I resolve these comments or hold on for now? Just checking in because I noticed that the "waiting for author" tag is still applied and I'm not sure if someone is currently waiting on me.

Reply:
I've come around to @Kenix3 's point of view here: let's name it based on how it's set, even if it's not used. It appears to be a boolean, too, so it should be `true`/`false` rather than `1`/`0`.